database/1-create-database.sql

```diff
@@ -0,0 +1,26 @@
+create database hotel;
+
+use hotel;
+
```

Comment:
Please create two sentences, one to delete booking table, and another to delete guest table.